include/freerdp/api.h

```diff
@@ -52,7 +52,7 @@
 #endif
 #endif
 #else
-#if __GNUC__ >= 4
+#if defined(__GNUC__) && (__GNUC__ >= 4)
```

Comment:
Just a question: Can we actually fall into this here? I.e. that, when `__GNUC__` is not defined, that this condition is evaluated here as `true`?

Reply:
@pnowack if you are not on windows and do not compile with `gcc`/clang` then yes